tools/dockerfile-llb-frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.20 AS builder
```

Comment:
Can you add a copyright preamble to the top of this file:
```Dockerfile
# SPDX-License-Identifier: BSD-3-Clause
# Copyright (c) 2022, NEC Europe Ltd., Unikraft GmbH, and The KraftKit Authors.
# Licensed under the BSD-3-Clause License (the "License").
# You may not use this file except in compliance with the License.
```

Reply:
Done